index.d/pipeline-images.yml

```diff
@@ -48,7 +48,7 @@
     job-id: pipeline-scanner
     git-url: git://github.com/dharmit/container-pipeline-service
     git-branch: master
-    git-path: pipeline_scanner
+    git-path: atomic_scanners/pipeline-scanner
```

Comment:
ideally yes, @dharmit was doing changes on this so thought may be we can build from his branch.